src/components/report-issue.tsx

```diff
@@ -51,6 +53,17 @@ export function ReportIssueForm({
       return;
     }
 
+    if (contactInfo.trim() === "") {
+      toast("Please enter your contact information.");
+      return;
+    }
```

Comment:
@adambard1 Just to confirm, we want to make this a required field? They can't report an issue unless they provide an email or phone number?